docs/installation.md

```diff
@@ -0,0 +1,170 @@
+# Installation
+
+Currently we support following ways, choose any of your comfort: 
+
+- [Power Shell](#power-shell) - easiest way
```

Comment:
`PowerShell` is right way of writing (without whitespace) _(need to correct everywhere in new files or changed lines in README)_